### PR TITLE
Fix/tickpricecalc

### PR DESCRIPTION
**Pull request by lnist**

### CODE REVIEW

File: src/util/tickPriceHelper.ts [hunk 1]
```diff
@@ -20,44 +27,111 @@ class TickPriceHelper {
...
+   * Calculates the price at a given raw offer list tick.
    * @param tick tick to calculate price for
-   * @returns price at tick
+   * @returns price at tick (not to be confused with offer list ratio).
    */
-  priceFromTick(tick: BigNumber): Big {
-    const priceFromTick = TickLib.priceFromTick(tick);
-    const p = Big(10).pow(
-      Math.abs(this.market.base.decimals - this.market.quote.decimals)
+  priceFromTick(tick: BigNumberish): Big {
```
Comment (Louis-Amas) [inline src/util/tickPriceHelper.ts:34]: Shouldn't it take tickSpacing ?

File: src/util/tickPriceHelper.ts [hunk 2]
```diff
@@ -20,44 +27,111 @@ class TickPriceHelper {
...
 
   /**
-   * Calculate the tick at a given price.
-   * @param ba bids or asks
+   * Calculates the raw offer list tick at a given order book price (not to be confused with offer list ratio).
    * @param price price to calculate tick for
-   * @returns tick for price
+   * @returns raw offer list tick for price
+   */
+  tickFromPrice(price: Bigish): BigNumber {
```
Comment (Louis-Amas) [inline src/util/tickPriceHelper.ts:60]: Shouldn't it take tickSpacing ?

**Review by dontrolle — APPROVED** [review]
(review offline)

File: src/util/tickPriceHelper.ts (hunk 1 above)
Comment (lnist) [reply src/util/tickPriceHelper.ts:34]: yes but not part of this PR

**Pull request merged**